components/scrubber/scrubber_test.go

```diff
@@ -471,6 +471,58 @@ func BenchmarkKeyValue(b *testing.B) {
 	}
 }
 
+// go test -bench=BenchmarkMetrics -benchmem -benchtime=5s
```

Comment:
How come this is commented out? ☺️

Reply:
have a comment below 😄 it's to prevent adding the prometheus dependency to this lib

Reply:
haha sorry, my eyes just skip over code that is commented out it seems :D